test/config/cassdc/cassdc-non-reaper.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: cassandra.datastax.com/v1beta1
```

Comment:
It looks like the only differences between cassdc-non-reaper.yaml and cassdc.yaml are the name and clusterName properties. We can avoid a lot of that duplication. I will work on an example of how best to do that.

Reply:
Also the size is different (again, to speed up the start and reduce resource usage). But yes, templating this would be a better option.